task/sql-tasks.js

```diff
@@ -178,7 +268,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            Count(ProductID) as "TotalOfCurrentProducts",
+            Sum(Discontinued=1) as "TotalOfDiscontinuedProducts"

```

Comment:
use count to show intent explicitly

but better to just do this


```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```
notice from-clause is optional